obspy/signal/spectral_estimation.py

```diff
@@ -1824,6 +1832,31 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 else:
                     xdata = periods
                 ax.plot(xdata, noise_model, '0.4', linewidth=2, zorder=10)
+        
+        if show_earthquakes:
+            earthquakes=get_earthquakes()
+            for eqk in earthquakes:
+                # Eq.1 Cauzzi & Clinton (2013)
+                ydata=[v/((1./earthquakes[eqk][0][iv])**(-.5)) for iv,v in enumerate(earthquakes[eqk][1])]
+                ydata=[20*np.log10(v/2) for v in ydata]
```

Comment:
spaces around arithmetic operators etc. please avoid cryptic 1-3 character variables, variable names should be meaningful, but with the above comment that should be fixed anyway.

Reply:
Ok.